modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv

```diff
@@ -370,7 +370,7 @@
 5.0674,3.9551,4.6865,3.9281,3.783,5.0467,3.9821,3.5421,3.7674,3.9582,5.7088,2.9829,3.0135,5.1943,4.1577,5.6793
 5.067,3.9555,4.6882,3.9284,3.7856,5.0398,3.9817,3.5421,3.7672,3.9527,5.7017,2.9829,3.0125,5.1967,4.1573,5.6637
 5.0666,3.9559,4.6899,3.9287,3.7883,5.0329,3.9813,3.5423,3.7669,3.9473,5.6945,2.9829,3.0116,5.1992,4.157,5.6484
-5.0662,3.9563,4.6916,3.9291,3.791,5.0261,3.9809,3.5426,3.7665,3.942,5.6873,2.9827,3.0109,5.2017,4.1568,5.6332
```

Comment:
Are you sure to replace the value in this curves file ?

Reply:
Yes. It does not influence the lidar. Each lidar has its own calibration file, so this file will be replaced when users bring up their own lidar.